test/dune

```diff
@@ -9,9 +8,8 @@
 
 (executable
  (name RunTests)
- (public_name RunTests.exe)
- (package brisk-reconciler)
- (libraries brisk-reconciler.test)
+ (libraries BriskReconcilerTest)
+ (preprocess (pps brisk_jsx ppx_deriving.std))
```

Comment:
while we're on it, let's change the ppx name to brisk_ppx instead of brisk_jsx, shall we?

Reply:
Done